bika/lims/browser/templates/stickers_preview.pt

```diff
@@ -142,7 +159,10 @@
                 window.location = $(this).attr('data-url');
             });
             $('select#template').change(function(e) {
-                reload($('select#template').val());
+                reload($('select#template').val(), $('#copies_count').val());
+            });
+            $('#copies_count').change(function(e) {
+                update_sticker_copies($('#copies_count').val(), $('select#template').val());
```

Comment:
Instead of a new function, reuse `reload`, it already provides the functionality you need:
```javascript
$('#copies_count').change(function(e) {
    reload($('select#template').val(), $('#copies_count').val());
});
```